### ISSUE TITLE
Chrome does not download the file viewer.properties (localized l10n file)

### ISSUE THREAD

**Issue by arielcarrera** [mozilla/pdf.js#6048]
When I open the demo or my custom implementation... I see in console:

#thumbs is undefined.
l10n.js:901 #thumbs is undefined.
l10n.js:829 #thumbs_label is undefined.
l10n.js:901 #thumbs_label is undefined.
l10n.js:829 #outline is undefined.
l10n.js:901 #outline is undefined.
l10n.js:829 #outline_label is undefined.
l10n.js:901 #outline_label is undefined.
l10n.js:829 #attachments is undefined.
l10n.js:901 #attachments is undefined.
l10n.js:829 #attachments_label is undefined.
l10n.js:901 #attachments_label is undefined....
...

Chrome does not download the file viewer.properties
I tested it on FF and It works ok but in Chrome it doesn't work.

**Comment by Rob--W** [comment]
Can't reproduce. Could you share a link to a broken example?

![pdfjsissue6048](https://cloud.githubusercontent.com/assets/1365071/8016070/0a4b0998-0bdf-11e5-8d03-232393c66736.png)

**Comment by timvandermeij** [comment]
Closing as incomplete until the information mentioned above is provided.

**Comment by aplanos** [comment]
I ran with the same and found the solution by adding a new import in the locale.properties file. When I was debugging I found that the locale variable in my browser was getting initialized with 'es' value. When I open the locale.properties file there were not a match for this value, so, i add it, referencing the es-es property file like this: 

[es]
@import url(es-ES/viewer.properties)

This works for me.
